lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -109,5 +150,8 @@ function gutenberg_enqueue_global_styles() {
 remove_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_global_styles_assets' );
 remove_action( 'wp_footer', 'gutenberg_enqueue_global_styles_assets' );
 
+// Enqueue global styles, and then block supports styles.
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_global_styles' );
 add_action( 'wp_footer', 'gutenberg_enqueue_global_styles', 1 );
+add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_stored_styles' );
+add_action( 'wp_footer', 'gutenberg_enqueue_stored_styles', 1 );
```

Comment:
Thanks @andrewserong Good catch. 

I've now added a similar check for global styles but without the `wp_should_load_separate_core_block_assets`, which I think we don't yet need since we're registering block supports and misc styles.